hazelcast/src/main/java/com/hazelcast/client/impl/protocol/util/ClientMessageAccumulator.java

```diff
@@ -0,0 +1,63 @@
+package com.hazelcast.client.impl.protocol.util;
+
+import com.hazelcast.client.impl.protocol.ClientMessage;
+
+import java.nio.ByteBuffer;
+
+/**
+ * ClientMessage builder from incoming bytebuffer parts
+ */
+public class ClientMessageAccumulator
```

Comment:
ClientMessageAccumulator is a ClientMessage with accumulation feature. Maybe we can update its name.